components/Card/index.js

```diff
@@ -0,0 +1,49 @@
+import styled from 'styled-components';
+
+const StyledCard = styled.div`
+  background-color: #f9f9f9;
+  border-radius: 10px;
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.1);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 2rem auto;
+  max-width: 800px;
+  overflow: hidden;
+`;
+
+const StyledLocation = styled.div`
+  background-color: #ffffff;
+  border-radius: 10px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  flex-basis: 50%;
+  padding: 1.5rem;
+`;
+
+const StyledLabel = styled.h3`
+  color: #666666;
+  font-size: 1rem;
+  margin-bottom: 1rem;
+  text-transform: uppercase;
+`;
+
+const StyledData = styled.div`
```

Comment:
how about using a p? It is more semantic:

```suggestion
const StyledData = styled.p`
```